src/common/models/Projections.ts

```diff
@@ -8,6 +8,12 @@ import { LocationSummary, MetricSummary } from 'common/location_summaries';
 import { RegionSummaryWithTimeseries } from 'api/schema/RegionSummaryWithTimeseries';
 import { County, Region } from 'common/regions';
 
+export type MetricValue = number | null;
```

Comment:
Hrm, not sure this is a helpful abstraction.  `MetricValue` doesn't immediately imply to me that it's a nullable value and so I think the code may actually be clearer just sticking with `number | null` rather than introducing extra indirection here.  So I'd prefer drop this type.

Reply:
fair point.  I was trying to simplify the typing, because of having to pass it for props in a couple places.  But I can revert this bit here.